public/local/en.json

```diff
@@ -35,5 +35,9 @@
   "ACCEPT_AND_TERMS_6": "We certify that the benefit program does not discriminate based on race, gender, religion, or any other characteristic protected by law. We commit to ensuring that all eligible beneficiaries are treated fairly throughout the application and selection process.",
   "ACCEPT_AND_TERMS_7": "We understand that any personal information collected from beneficiaries during the application process will be kept confidential and used solely for the purpose of evaluating and administering the benefit program. We agree to comply with all relevant data protection and privacy laws.",
   "ACCEPT_AND_TERMS_8": "We acknowledge that amendments or updates to the benefit details may be required, and we agree to make such changes transparently and promptly. We understand that any significant changes will be communicated to potential beneficiaries in a timely manner.",
-  "ACCEPT_AND_TERMS_9": "By publishing this benefit to the public domain, our organization agrees to abide by these terms and conditions and to ensure that the benefit program is managed in a transparent, fair, and lawful manner."
+  "ACCEPT_AND_TERMS_9": "By publishing this benefit to the public domain, our organization agrees to abide by these terms and conditions and to ensure that the benefit program is managed in a transparent, fair, and lawful manner.",
+  "LOGIN_ERROR": "Provider with this email does not exist, try registering",
```

Comment:
1. add new constants in this format - `SCREEN/ENTITY-NAME_ACTION/LABEL`
```
LOGIN_ERROR
LOGIN_OTP_ERROR
REGISTER_ERROR
```

2 - add alphabetically